components/places/ios/Places/HistoryMetadata.swift

```diff
@@ -28,6 +43,16 @@ public enum HistoryMetadataObservation {
     case titleObservation(String)
     case viewTimeObservation(Int32)
     case documentTypeObservation(DocumentType)
+    case unknown
```

Comment:
What is an `unknown` observation, and why do you need it? Does seem necessary to me.

Reply:
I really don't need unknown but removing that causes swift errors for our initializers so I added additional case for default as unknonw. Happy to change / rename it as needed.

Reply:
What about function overloading? Separate init functions for each data type? Then you can remove unknown, i think.